orchagent/portsorch.cpp

```diff
@@ -4313,13 +4324,16 @@ bool PortsOrch::addBridgePort(Port &port)
         }
     }
 
+    port.m_bridge_port_admin_state = true;
+
     if (!setHostIntfsStripTag(port, SAI_HOSTIF_VLAN_TAG_KEEP))
     {
         SWSS_LOG_ERROR("Failed to set %s for hostif of port %s",
                 hostif_vlan_tag[SAI_HOSTIF_VLAN_TAG_KEEP], port.m_alias.c_str());
         return false;
     }
     m_portList[port.m_alias] = port;
+    portOidToName[port.m_bridge_port_id] = port.m_alias;
```

Comment:
Are we saving both port object id to alias an bridge pot id to alias in data structure? It also has vlan and lag. Shouldn't we rename this to something like saiOidToAlias?

Reply:
Yes. will change the name to saiOidToAlias.